docs/github/continuous-integration.md

```diff
@@ -0,0 +1,284 @@
+
+Here are some of the benefits of using PRs:
+
+- **Collaboration**: PRs allow developers to collaborate on code and share ideas.
+- **Code review**: PRs allow other developers to review your changes and provide feedback.
+- **Testing**: PRs can be used to test changes before they are merged into the main codebase.
+- **Documentation**: PRs can be used to document changes to the code.
+- **History**: PRs provide a history of changes to the code.
+
+## Creating or switching branches
+
+1. Go to your Repository > https://github.com/cagov/[repsoitory_name]
+2. Click on the Branch Dropdown
+    1. On the main page of your repository, you'll see a dropdown menu on the top left, displaying the current branch (usually “main”). Click on this dropdown.
+3. Type or click on an Branch Name
+    1. In the dropdown that appears, there’s a textbox at the top, type the name for the branch you want to switch to or create.
+    2. When creating a new branch, choose a descriptive and relevant name. e.g. `project_staging_model`
+    3. **If you’re just switching branches you are done after this step!**
+4. Choose a base for the New Branch
+    1. Usually this is “main” as you want to work off of the most up-to-date version of code, but you can also choose another branch to base your new one off of if you want to add to someone else’s work in progress without making edits directly to their branch and causing a merge conflict! 😬
+5. Click "Create branch `<branch_name>` from `<base_branch_name>`" (usually your base branch will be “main”)
+
+Now, you have successfully created a new branch in your GitHub repository and will be automatically switched to this branch. You can make changes on this new branch without affecting the main branch.
+
+If you're working with git locally, you can create and switch branches as well as check your current git branch using the following git commands:
+
+- **Create and switch to a branch:** `git switch -c <branch_name>`
+    - **This also works, but is the older way of doing it:** `git checkout -b <branch_name>`
+- **Switch to an existing branch:** `git switch <branch_name>`
+    - If this doesn’t work it’s likely that you have to pull down the remote branch you want to work with first using `git fetch`
+    - Then you can run git switch `<branch_name`> again
+
+🛎️Note: replace `<branch_name>` with the actual name of your new branch e.g. `writing_project_docs`.
+
+- Check which branch you are on: `git branch --show-current`
+- Optional – persistently show your full path and branch name in command line:
+
+**Mac OS**
+
+- Type `open ~/.bash_profile`
+- In the file that opens, add the following at the bottom of the file:
+
+    `parse_git_branch() {
+    git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/ (\1)/'
+    }
+    export PS1="\u@\h \[\033[32m\]\w -\$(parse_git_branch)\[\033[00m\] $ "`
+
+**Windows OS (Linux-based)**
```

Comment:
I'm not sure what you mean by Linux-based here. Do you mean using git-bash? Or WSL?

Reply:
This was pulled from @britt-allen's  original documentation found [here](https://docs.google.com/document/d/1mv-0hymSr2WIUgE-9l6P53M44Fd13nQ_QUpgxSIgKTU/edit?tab=t.0#heading=h.ds8m1tvllmwd)

Reply:
(Linux-based)  has been removed.